### PR TITLE
🌱  Remove Calico static files

### PR DESCRIPTION
**Pull request by fmuyassarov**
**What this PR does / why we need it**:
We don't need these static files of Calico manifests since after https://github.com/metal3-io/cluster-api-provider-metal3/pull/277 we are now pulling the manifests from upstream and fetching them on the fly.

### CODE REVIEW

**Review by kashifest — COMMENTED** [review]
Is there any reason we are hard-coding this? Why can't we inherit this from m3-dev-env?

**Review by kashifest — APPROVED** [review]

**Comment by kashifest** [comment]
/hold
Until prow and metal3-dev-env is tracking the main branch for CAPM3 in integration tests

**Comment by fmuyassarov** [comment]
/close
I am not able to spent time on this right now.

File: test/e2e/e2e_suite_test.go [hunk 1]
```diff
@@ -158,7 +158,6 @@ func createClusterctlLocalRepository(config *clusterctl.E2EConfig, repositoryFol
 	// Ensuring a CNI file is defined in the config and register a FileTransformation to inject the referenced file as in place of the CNI_RESOURCES envSubst variable.
 	Expect(config.Variables).To(HaveKey(capi_e2e.CNIPath), "Missing %s variable in the config", capi_e2e.CNIPath)
 	cniPath := config.GetVariable(capi_e2e.CNIPath)
-	Expect(cniPath).To(BeAnExistingFile(), "The %s variable should resolve to an existing file", capi_e2e.CNIPath)
```
Comment (namnx228) [inline test/e2e/e2e_suite_test.go:161]: @furkatgofurov7 The thing that you need to modify should be around here.

**Pull request closed without merging**